src/Tools/ExternalAccess/Razor/Cohost/RazorCohostLanguageClient.cs

```diff
@@ -60,9 +60,9 @@ public override ServerCapabilities GetCapabilities(ClientCapabilities clientCapa
         Contract.ThrowIfNull(razorCapabilitiesProvider);
 
         // We use a string to pass capabilities to/from Razor to avoid version issues with the Protocol DLL
-        var serializedClientCapabilities = JsonConvert.SerializeObject(clientCapabilities);
+        var serializedClientCapabilities = JsonSerializer.Serialize(clientCapabilities);
         var serializedServerCapabilities = razorCapabilitiesProvider.GetCapabilities(serializedClientCapabilities);
-        var razorCapabilities = JsonConvert.DeserializeObject<VSInternalServerCapabilities>(serializedServerCapabilities);
+        var razorCapabilities = JsonSerializer.Deserialize<VSInternalServerCapabilities>(serializedServerCapabilities);
```

Comment:
This whole file is unused, so this change was quite unnecessary, but figured it would be nicer to paint a clearer picture. https://github.com/dotnet/roslyn/pull/73040 removes the file entirely, just have to wait for VS Preview to ship before we can merge.